automated install/basic-install.sh

```diff
@@ -955,6 +964,9 @@ accountForRefactor() {
 	sed -i 's/piholeDNS1/PIHOLE_DNS_1/g' ${setupVars}
 	sed -i 's/piholeDNS2/PIHOLE_DNS_2/g' ${setupVars}
 
+	# Account for change in lighttpd config file
+	sed -i 's:pihole/index.html:pihole/index.php:' ${lighttpdConfig}
```

Comment:
Shouldn't the config get overwritten on line 715, making it already have the right path?